sdk/recoveryservices/azure-mgmt-recoveryservicesbackup/azure/mgmt/recoveryservicesbackup/activestamp/_patch.py

```diff
@@ -24,8 +24,40 @@
 # IN THE SOFTWARE.
 #
 # --------------------------------------------------------------------------
+from typing import List
+import importlib
+import urllib.parse
+from ._recovery_services_backup_client import RecoveryServicesBackupClient as RecoveryServicesBackupClientGenerated
+from azure.core.pipeline.policies import SansIOHTTPPolicy
+
+class RemoveDuplicateParamsPolicy(SansIOHTTPPolicy):
+    def __init__(self, duplicate_param_names):
+        # type: (List[str]) -> None
+        self.duplicate_param_names = duplicate_param_names
+
+    def on_request(self, request):
+        parsed_url = urllib.parse.urlparse(request.http_request.url)
+        query_params = urllib.parse.parse_qs(parsed_url.query)
+        filtered_query_params = {
+            k: v[-1:] if k in self.duplicate_param_names else v
+            for k, v in query_params.items()
+        }
+        request.http_request.url = request.http_request.url.replace(parsed_url.query, "") + urllib.parse.urlencode(filtered_query_params, doseq=True)
+        return super().on_request(request)
+
+DUPLICATE_PARAMS_POLICY = RemoveDuplicateParamsPolicy(duplicate_param_names=["$filter", "$skiptoken", "api-version"])
+
+class RecoveryServicesBackupClient(RecoveryServicesBackupClientGenerated):
+    def __init__(self, credential, *args, **kwargs):
```

Comment:
We have to make `credential` clear because [azure-cli](https://github.com/Azure/azure-cli/blob/67568014e3df1f183b92ee1ad7699b3b86f35dc0/src/azure-cli-core/azure/cli/core/util.py#L631-L637).

Reply:
Let me update the other client too